### DISCUSSION TITLE
Shield default path doesn't work

### DISCUSSION THREAD

**Discussion by AlbertoMoricca** [codeigniter4/shield#538] [Q&A] [answered]
Hi,

Basing on my understanding, the default registration/login path of Shield framewok don't work. For example, the "/register" path doesn't open the registration form. Then, I can't use the default function of Shield (example: the email activator). What's the issue?

Thanks,
Alberto

**Comment by datamweb** [comment, marked as answer]
Have you run command 
```console
php spark shield:setup
```
after installation?

**Reply by AlbertoMoricca** [reply]
Yes, i did.

**Reply by datamweb** [reply]
What error do you get in environment development when you request page login?

**Reply by AlbertoMoricca** [reply]
The error is "File not found" (I'm trying with register). Following a screen:

![immagine](https://user-images.githubusercontent.com/105434937/203593347-7467f9f5-d82d-4d24-8d0d-7d23c9d26efe.png)

Can be my custom routes an issue? The routes are the following:

`$routes->get('/index.php', 'Pages::view');
$routes->get('/', 'Pages::view');
$routes->get('/mongodb', 'MongoDB::view');
$routes->get('/mysql', 'Mysql::view');
$routes->get('/provaemail', 'TestEmail::view');
$routes->get('/test', 'TestController::view');
$routes->post('/(:any)', 'PostController::$1');
$routes->get('/logout', 'LogoutController::logout');
$routes->get('/admin/(:any)', 'Pages::view_admin/$1');
$routes->get('(:any)', 'Pages::view/$1');`

At any cases, I disable all of them and the error is a 404 with "route not found":

![immagine](https://user-images.githubusercontent.com/105434937/203593763-9d556948-3ab0-4e7e-8d8c-1d72eabea0b1.png)

**Reply by datamweb** [reply]
Output ?
```console
php spark routes

```

**Reply by AlbertoMoricca** [reply]
![immagine](https://user-images.githubusercontent.com/105434937/203595717-0951c77f-833f-4a3d-a272-fa231af365d2.png)

**Reply by datamweb** [reply]
As it is clear, you have not installed the shield correctly.
Please follow the installation instructions. Also, Shield requires version 4.2.7 and above CI, you are using version 4.2.6.
Use the latest version of CI and also the latest version of Shield.

https://github.com/codeigniter4/shield/blob/develop/docs/install.md

**Reply by AlbertoMoricca** [reply]
Ok, I tried to perform a clean installation of Codeigniter and Shield and the routes are correctly set. I compared the code between the two installation and the issue is the missing row in the App/Routes.php:

service('auth')->routes($routes);

Maybe doesn't the edit on the routes.php work if the routes is not the original routes.php? Anyway, thanks for the help!

**Reply by kenjis** [reply]
> Maybe doesn't the edit on the routes.php work if the routes is not the original routes.php?

No, but it is possible that there is a bug that is preventing the setup command from working. 
If you have a `Routes.php` that is not edited, please file a bug report.

**Reply by AlbertoMoricca** [reply]
Ok, I will do this. Thanks a lot.
